testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewGoldenTestManager.m

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "PlatformViewGoldenTestManager.h"
+
+@interface PlatformViewGoldenTestManager ()
+
+@property(readwrite, strong, nonatomic) GoldenImage* goldenImage;
+@property(readwrite, strong, nonatomic) NSArray* launchArgs;
+
+// Maps the `identifier` to the launch args for the platform view test associated with this
+// `GoldenImage`
+@property(class, nonatomic, strong) NSDictionary* launchArgsMap;
+
+@end
+
+@implementation PlatformViewGoldenTestManager
+
+static NSDictionary* _launchArgsMap;
+
++ (void)initialize {
```

Comment:
Overriding `+initialize` is very frowned upon.  You can make the static dictionary a local variable in -initWithIdentifier.  Or create the tested golden out of the launch args.  Or make it an `extern` somewhere to be used by AppDelegate as @dnfield suggests.

Reply:
> Or make it an `extern` somewhere to be used by AppDelegate as @dnfield suggests.

@cyanglaz correctly points out this isn't possible in a XCUITest.

Reply:
Ahh right.